CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ which use the proxied NSX-V API of advanced edge gateway for handling NAT rules
 * Added methods `Client.GetVMByHref` `Vapp.GetVAMByName` and related `GetVMById`, `GetVAMByNameOrId`
 * Deprecated methods `Client.FindVMByHREF`, `Vdc.FindVMByName`, `Vdc.FindVAppByID`, and `Vdc.FindVAppByName`
 * Added methods `Vm.GetGuestCustomizationSection` and `Vm.SetGuestCustomizationSection`  
+* Converted private method `getBareEntityUuid` to a public `GetBareEntityUuid`
+* Added new method `QueryMediaImage`
```

Comment:
I see a new Query method, but I was expecting a set of `Get*` methods like we have done in the other entities for which we provide data sources.
I'd like to see uniformity, and have methods
`Catalog.GetMediaImageByName`,  `Catalog.GetMediaImageById`, `Catalog.GetMediaImageByNameOrId`
as wrappers around the current methods.
When I look for a media image, I want to know that there is a method named in a consistent manner that gives me the expected results.

Moreover, we need to deprecate `FindMediaImage` and replace all its usage  everywhere

Reply:
They were added just recently I believe, @vbauzysvmware ?

@vbauzysvmware also after that recent commit this should state `QueryMedia` not `QueryMediaImage` anymore? Please check for more places where old "QueryMediaImage" name might have remained unchanged.